### ISSUE TITLE
Several kwargs are no longer supports by konch 5.0.0

### ISSUE THREAD

**Issue by jdimmerman** [sloria/flask-konch#41]
Hello! Thank you for the tool.

As of konch 5.0.0, the following kwargs aren't supported and result in exceptions when invoking flask konch

```
KONCH_IPY_AUTORELOAD: Whether to load and enable the IPython autoreload extension (must be using ipython shell).
KONCH_IPY_EXTENSIONS: List of IPython extension names to load (must be using ipython shell).
KONCH_IPY_COLORS: IPython color style.
KONCH_IPY_HIGHLIGHTING_STYLE: IPython code highlighting style.
KONCH_PTPY_VI_MODE: Enable vi mode (must be using ptpython shell).
```

**Comment by sloria** [comment]
oops! this is actually a regression in konch itself. I've released a bugfix, so this should be fixed by upgrading to konch 5.0.1.

thanks for reporting this!

**Comment by jdimmerman** [comment]
Awesome, thank you @sloria!
